docs/test.ts

```diff
@@ -115,7 +115,6 @@ tracer.init({
     },
     apiSecurity: {
       enabled: true,
-      requestSampling: 1.0
```

Comment:
breaking change :/

Reply:
We checked the values being used, and no one is using a custom sampling rate only the default. It's safe to remove it.